src/main/java/com/easypost/model/Shipment.java

```diff
@@ -548,6 +550,42 @@ public void setTaxIdentifiers(final List<TaxIdentifier> taxIdentifiers) {
         this.taxIdentifiers = taxIdentifiers;
     }
 
+    /**
+     * Get the carrier accounts list.
+     *
+     * @return List of carrier accounts.
+     */
+    public List<CarrierAccount> getCarrierAccounts() {
+        return carrierAccounts;
+    }
+
+    /**
+     * Set the carrier accounts list.
+     *
+     * @param carrierAccounts the list of carrier accounts.
+     */
+    public void setCarriersAccount(final List<CarrierAccount> carrierAccounts) {
```

Comment:
Should this be named `setCarrierAccounts` ?

Reply:
nice catch!